react/features/base/dialog/actions.js

```diff
@@ -0,0 +1,32 @@
+import {
+    BEGIN_DIALOG_REQUEST,
+    END_DIALOG_REQUEST
+} from './actionTypes';
+
+/**
+ * Signals Dialog to close its dialog.
+ *
+ * @returns {{
+ *     type: END_DIALOG_REQUEST
+ * }}
+ */
+export function hideDialog() {
+    return {
+        type: END_DIALOG_REQUEST
```

Comment:
Yep, that's the reason to have the container so we can play with multliple dialogs if need to. I think I have seen this in current meet.